src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -1668,12 +1690,56 @@ else if (v instanceof ListValue)
             }
         });
 
+        put("blue_skull",(e,v)->{
+            if (e instanceof WitherSkull w){
+                w.setDangerous(v.getBoolean());
+            }
+            
+        });
+        put("offering_flower",(e,v)->{
+            if (e instanceof IronGolem ig){
+                ig.offerFlower(v.getBoolean());
+            }
+            
+        });
+        put("item", (e, v) -> {
+                ItemStack item=getItemStackFromValue(v);
+                if(e instanceof ItemEntity itementity)            
+                    itementity.setItem(item);
+                if(e instanceof ItemFrame itemframe)
+                    itemframe.setItem(item);  
+        });
         // "dimension"      []
-        // "item"           []
         // "count",         []
         // "effect_"name    []
     }};
 
+    static public ItemStack getItemStackFromValue(Value value){
+        List<Value> items;
+        if(value instanceof ListValue blv && blv.length()==3){
+            items = blv.getItems();
+        }else if(value.isNull()){
+            return ItemStack.EMPTY;
+        }else{
+            items= List.of(StringValue.of(value.getString()),Value.ONE,Value.NULL);
```

Comment:
this seems little excessive, if only string value is provided, then you could grab the item from registries and make a single stack out of it. check crafting_remaining_item and then item.getDefaultInstance()

Reply:
ok, for the sake of completeness, I guess it works with 
`item_display_name('iron_nugget{RepairCost:0,display:{Name:\'{"text":"IroNug"}\'}}')`
it does which is nice, but having a shortcut of submitting just registry valid name makes sense to me from the performance perspective

Reply:
ok, since NBTSerializableValue.parseItem already does that disregard this comment and use that instead.